portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -485,4 +513,27 @@ definition {
 		AssertElementPresent(locator1 = "WorkflowMetrics#METRICS_WORKLOAD_BY_ASSIGNEE_PAGE_TASKCOUNT");
 	}
 
+	macro viewWorkloadByAssigneeSingleApprover {
+		var key_assigneeName = "${assigneeName}";
+
+		for (var isInstancePresent : list "${isPresent}") {
+			var key_assigneeName = "${isInstancePresent}";
+
+			AssertElementPresent(locator1 = "WorkflowMetrics#METRICS_WORKLOAD_BY_ASSIGNEE_NAME_TABLE");
+
+			AssertElementPresent(locator1 = "WorkflowMetrics#METRICS_WORKLOAD_BY_ASSIGNEE_TASKCOUNT_OVERDUE");
+
+			AssertElementPresent(locator1 = "WorkflowMetrics#METRICS_WORKLOAD_BY_ASSIGNEE_TASKCOUNT_ON_TIME");
+
+			AssertElementPresent(locator1 = "WorkflowMetrics#METRICS_WORKLOAD_BY_ASSIGNEE_TASKCOUNT_ON_TIME");
```

Comment:
Hi @calvinctvo ,
The same validation is being performed twice. Would you please take a look?